docs/tips/03-if-else-in-JSX.md

```diff
@@ -33,23 +33,23 @@ That's not valid JS. You probably want to make use of a ternary expression:
 React.render(<div id={condition ? 'msg' : ''}>Hello World!</div>, mountNode);
 ```
 
-If a ternary expression isn't robust enough, you can use `if` statements to determine which
-components should be used.
+You can also use ternary expression to determine which tags or components should be used.
```

Comment:
Agree with @zpao. Not sure about the new ternary below. I feel like an `if` is fine.